http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcRequestAuthenticator.java

```diff
@@ -221,8 +228,10 @@ protected String getRedirectUri(String state) {
 
             String redirectUri = rewrittenRedirectUri(url);
             URIBuilder redirectUriBuilder = new URIBuilder(deployment.getAuthUrl());
+            String cryptoValue = Oidc.getCryptographicValue(sessionRandomValue);
             redirectUriBuilder.addParameter(RESPONSE_TYPE, CODE)
-                    .addParameter(CLIENT_ID, deployment.getResourceName());
+                    .addParameter(CLIENT_ID, deployment.getResourceName())
+                    .addParameter(NONCE, String.valueOf(cryptoValue));
```

Comment:
Since cryptoValue is already a String, String.valueOf isn't needed.

Reply:
done